<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>50000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>F8Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="F8Tool.qrc">
    <normaloff>:/resources/images/icon.ico</normaloff>:/resources/images/icon.ico</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/f8filter.png);</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>521</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/resources/images/signin.png);</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>501</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Api Key:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_api_key">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dLtBejuUGo5gowWHQZnCnKntMqP1a068enKAzjRm</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_api_hash">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PwSW8NQORuIk2yrOXxY1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Api Hash:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="login_submit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="F8Tool.qrc"/>
 </resources>
 <connections/>
</ui>
